### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LonglonWu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1202,6 +1205,9 @@ IEEE Trans. Med. Imaging 40(11), 3229–3237 (2021).
 - R. K. Gupta, N. Hempler, G. P. A. Malcolm, K. Dholakia, and S. J. Powis    
 *[High throughput hemogram of T cells using digital holographic microscopy and deep learning](https://doi.org/10.1364/OPTCON.479857)*  
 Opt. Continuum 2(3), 670 (2023).
+- L. Wu, S. Bak, Y. H. Shin, Y. S. Chu, S. Yoo, I. K. Robinson and X. J. Huang    
+*[Resolution-enhanced X-ray fluorescence microscopy via deep residual networks](https://doi.org/10.1038/s41524-021-00644-z)*
```
Comment (kqwang) [inline README.md]: Hi, Prof. Wu, the DOI should be "https://doi.org/10.1038/s41524-023-00995-9
".

**Review by kqwang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LonglonWu) [reply README.md]: Yes, you are right. Thank you so much for your careful check. The doi has been updated.

**Review by LonglonWu — COMMENTED** [review]
Yes, you are right. Thank you so much for your careful check.

File: README.md (hunk 1 above)
Comment (kqwang) [reply README.md]: Merged. Thank you for your efforts. Looking forward to contacting you further.

**Comment by kqwang** [comment]
Merged

**Pull request merged**